modules/ossm-config-otel.adoc

```diff
@@ -0,0 +1,117 @@
+// Module included in the following assemblies:
+//
+// * service-mesh-docs-main/traces/ossm-distr-tracing-assembly.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="ossm-config-otel_{context}"]
+= Configuring {OTELName} with Service Mesh
+
+You can integrate {SMProductName} with {OTELName} to instrument, generate, collect, and export telemetry traces, metrics, and logs to analyze and understand your software's performance and behavior.
+
+.Prerequisites
+
+* {TempoOperator} is installed. See: link:https://docs.redhat.com/en/documentation/openshift_container_platform/4.16/html/distributed_tracing/distributed-tracing-platform-tempo#distr-tracing-tempo-install-web-console_dist-tracing-tempo-installing[Installing the Tempo Operator].
+* {OTELOperator} is installed. See: link:https://docs.redhat.com/en/documentation/openshift_container_platform/4.16/html/red_hat_build_of_opentelemetry/install-otel[Installing the Red Hat build of OpenTelemetry]
+* A TempoStack is installed and configured in a `TempoStack` namespace. See: link:https://docs.redhat.com/en/documentation/openshift_container_platform/4.16/html/distributed_tracing/distributed-tracing-platform-tempo#installing-a-tempostack-instance[Installing a TempoStack instance].
+* An Istio instance is created in the `Istio` resource.
+* An Istio CNI instance is created in the `Istio` resource.
+
+.Procedure
+
+. Navigate to the {OTELOperator} and install the `OpenTelemetryCollector` resource in the `istio-system` namespace:
++
+.Example OpenTelemetry Collector in `istio-system` namespace
+[source, yaml]
+----
+kind: OpenTelemetryCollector
+apiVersion: opentelemetry.io/v1beta1
+metadata:
+  name: otel
+  namespace: istio-system
+spec:
+  observability:
+    metrics: {}
+  deploymentUpdateStrategy: {}
+  config:
+    exporters:
+      otlp:
+        endpoint: 'tempo-sample-distributor.tempostack.svc.cluster.local:4317'
```

Comment:
'tempo-sample-distributor.tempo.svc.cluster.local:4317'

update this endpoint.